tests/integration/reqs/test_submit_multisigned.py

```diff
@@ -20,16 +22,16 @@
 from xrpl.wallet import Wallet
 
 FEE = get_fee(JSON_RPC_CLIENT)
+SEQ = get_next_valid_seq_number_sync(WALLET.classic_address, JSON_RPC_CLIENT)
```

Comment:
This shouldn't be a global because the effectiveness of the sequence is impacted by the test case - so if multiple tests were added to this file, they wouldn't be able to re-use the SEQ value. (Similarly the values that depend on SEQ should be defined within the test case in my opinion) 

I think it's also a bit more readable to have all the moving parts in the test case, rather than assuming people read the top-level code before diving into the specific test they care about.